### ISSUE TITLE
Ignoring Palindrom's meta data

### ISSUE THREAD

**Issue by warpech** [Starcounter/starcounter-include#37]
https://github.com/Starcounter/level1/issues/4017 introduces JSON namespaces at the root level, which enables us to do blending at the root level.

There is a challenge though, because the root level also includes Palindrom meta data (`_ver#s`, `_ver#c$`).

Potential solutions proposed so far:

1. assume that namespaces are the keys which have a value of type `object`
2. ignoring `_.*`
3. wrap versions &co. in `_meta: {ver#s: 1, ver#c$: 3, something: 'blah'}`

**Comment by warpech** [comment]
4. Another solution: Only merge namespaces that have a value of type `object` and a `Html` property inside of that. Related: https://github.com/Starcounter/starcounter-include/issues/12

^ but this contradicts https://github.com/Starcounter/starcounter-include/issues/33

^^ and was rejected in https://github.com/Starcounter/starcounter-include/issues/11, I am not sure if correctly

**Comment by tomalec** [comment]
I really find 1. and 4. very limiting and hacky, see my comments in #11, #12, #33. Also we could solve original issue, without a need to change current design (less bugs, less breaking changes).

That's why I would go for 2. and/or 3. As it explicitly shows what's going on. 
"An `object` that has `Html` property" is a hard patter to find out by looking at root of JSON tree. While `_meta` will appear only on root level, will contain some Starcounter stuff and will never ever happen for regular app.

**Comment by tomalec** [comment]
> ^^ and was rejected in #11, I am not sure if correctly

Please share you doubts there.

**Comment by tomalec** [comment]
At [Blending TF meeting](https://github.com/Starcounter/Minutes/blob/e3a583d4a99da8f042fa33f7542da515c690fb76/BlendingTF/2017-07-05.md) we agreed to implement quick fix  by ignoring hard-coded Starcounter specific paths `_ver#s` and _`ver#c$` in `starcounter-include` to unblock  https://github.com/Starcounter/level1/issues/4061.
Then we will continue more solid discussion and implementation of ignored properties in future releases.

**Comment by tomalec** [comment]
@warpech Do you think we should merge it to Starcouner via separate PR or wait for https://github.com/Starcounter/level1/issues/4017?

**Comment by warpech** [comment]
Wait for https://github.com/Starcounter/level1/issues/4017.

**Comment by tomalec** [comment]
Let's close it as "frozen" - to be continued once we have time and resources.
